StylaSEO/Components/Styla/Utils.php

```diff
@@ -16,6 +16,39 @@ public static function getJsEmbedCode($username, $js_url = null){
 	    return '<script type="text/javascript" src="'.$url.'" async></script>';
     }
 
+    public static function createTag($tagObj) {
+        if ($tagObj->tag == "title" || $tagObj->tag == "script") {

```

Comment:
meta and link should not be closed. We have a logic here in seoapi:

    if (type === 'link' || type === 'meta') {
        return `<${type} ${attributeList.join(' ')}>`;
    }